core/src/play/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceFlavorHelper.java

```diff
@@ -151,7 +151,6 @@ public void onDisconnected() {
                 // hardware volume buttons control the local device volume
                 mediaRouter.setMediaSessionCompat(null);
                 unregisterWifiBroadcastReceiver();
-                callback.setupNotification(false, info);
```

Comment:
Are you sure that this is no longer needed?

Reply:
I am pretty sure - the rewrite in this PR is to simplify service start / stop and the associated notification, so that such calls do not need to be sprinkled in the codes.

Conceptually, the logic is streamlined so that the service start / stop / notification is simply dependent on whether there is media playback state, managed by `ServiceManager`. `ServiceManager` is the only one that will set the notification (accordingly dependent on the state).